tests/openvino/test_diffusion.py

```diff
@@ -130,30 +139,38 @@ def test_num_images_per_prompt(self, model_arch: str):
     @parameterized.expand(SUPPORTED_ARCHITECTURES)
     @require_diffusers
     def test_compare_to_diffusers_pipeline(self, model_arch: str):
-        height, width, batch_size = 128, 128, 1
+        height, width, batch_size = 64, 64, 1
         inputs = self.generate_inputs(height=height, width=width, batch_size=batch_size)
-
+        auto_cls = self.get_auto_cls(model_arch)
         ov_pipeline = self.OVMODEL_CLASS.from_pretrained(MODEL_NAMES[model_arch])
-        diffusers_pipeline = self.AUTOMODEL_CLASS.from_pretrained(MODEL_NAMES[model_arch])
-
-        for output_type in ["latent", "np", "pt"]:
-            inputs["output_type"] = output_type
+        diffusers_pipeline = auto_cls.from_pretrained(MODEL_NAMES[model_arch])
 
-            ov_output = ov_pipeline(**inputs, generator=get_generator("pt", SEED)).images
-            diffusers_output = diffusers_pipeline(**inputs, generator=get_generator("pt", SEED)).images
+        with torch.no_grad():
```

Comment:
I think this not necessary ? all diffusers call methods have no_grad decorators

Reply:
https://github.com/huggingface/diffusers/blob/main/src/diffusers/pipelines/sana/pipeline_sana.py#L673

Reply:
yes, you are right, I added it during debugging accuracy issue